cli/tests/pcluster/config/test_update_policy.py

```diff
@@ -1465,6 +1465,133 @@ def test_condition_checker_managed_placement_group(
             "If you intend to proceed with the update, please make sure to back-up your data and explicitly replace "
             "the file system(s) ({'test-fsx-lustre'}) with a new one(s) in the cluster configuration.",
         ),
+        # If change includes addition of SubnetIds and existing + new cluster configuration uses the same
+        # managed Fsx for Lustre
+        #   - Show Managed Fsx validation failure message
+        (
+            {
+                "Scheduling": {"Queues": [{"Name": "mock-q", "Networking": {"SubnetIds": ["subnet-12345678"]}}]},
+                "SharedStorage": [
+                    {"MountDir": "/test-fsx-lustre", "Name": "test-fsx-lustre", "StorageType": "FsxLustre"}
+                ],
+            },
+            {
+                "Scheduling": {"Queues": [{"Name": "mock-q", "Networking": {"SubnetIds": ["subnet-87654321"]}}]},
```

Comment:
This target configuration does not "add" a new subnet.
Expected:
```
"SubnetIds": ["subnet-12345678", "subnet-87654321"]
```

The test passes because technically there's a subnet change. But we should probably keep the test conditions as accurate as possible.

Reply:
thanks, I will keep the config aligned to the Change object